admin/src/scenes/volontaires-responsible/index.js

```diff
@@ -1,10 +1,10 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
```

Comment:
Let's keep it without `{ useEffect, useState }` since it's not used

```suggestion
import React from "react";
```